docs/source/guides/torii-tls.rst

```diff
@@ -0,0 +1,51 @@
+Configure TLS for client-peer communication (torii)
+===================================================
+By default, client-peer communication is not encrypted.
+To enable it, you need to:
+
+1. `Generate <Generating keys>`_ a key/certificate pair for each peer
+2. Distribute the certificate to all clients
+3. `Configure <Configuring irohad>`_ irohad to use these keys
+4. [Re]start irohad
+
+
+Generating keys
+~~~~~~~~~~~~~~~
+
+Keys must be presented in PEM format. To generate them you can use ``openssl``:
+
+```bash
+$ openssl genpkey -algorithm <desired algorithm> -out server.key
+$ openssl req -new -key server.key -x509 -out server.crt
+```
+
+If you need to use plain IP addresses to connect to the node, you need to
+specify subjectAltName in your server certificate, for that you need to add
+a ``subjectAltName`` directive to ``v3_ca`` section of your openssl config 
```

Comment:
Please install some plugin to your editor that shows or automatically removes trailing whitespaces.
```suggestion
You can enable TLS on torii ports if you would like to encrypt client-peer
```

Reply:
Fixed in latest version